lua/nvim-web-devicons-light.lua

```diff
@@ -285,6 +285,18 @@ local icons_by_filename = {
     cterm_color = "25",
     name = "Dockerfile",
   },
+  ["build"] = {
```

Comment:
These are very generic names.

It's not going to interfere with, say, meson build as that is a directory.

Let's add and see if anyone complains.